README.md

```diff
@@ -0,0 +1,21 @@
+# New XKit Organization
+
+New XKit is the community that has taken over the operation and maintenence of the XKit extension framework for Tumblr.
+
+This is the organization repository for collecting administrative information and records associated with the New XKit project.
+
+# Community
+
+The New XKit community is spread across various services.  Below is a list of official communication channels and accounts.
+
+- [GitHub](https://github.com/new-xkit)
```

Comment:
I'll go ahead and remove the link.  If they've found the Org README, GtiHub already gives them an easy navigation back to the Org.